client.py

```diff
@@ -77,14 +84,16 @@ def inner(feed=WDT_CB):
                             elif feed == WDT_CB and run:  # Timer callback and is running.
                                 cnt -= 1
                                 if cnt <= 0:
-                                    reset()
+                                    machine.reset()
```

Comment:
There was no reference to a reset() so I assumed that you meant machine.reset() or forgot the import statement.

Reply:
Well spotted.